### ISSUE TITLE
Custom element doesn't exist: fan-percent-button-row

### ISSUE THREAD

**Issue by bwitsken69** [finity69x2/fan-percent-button-row#45]
Here is my code and i get the error above.  Not sure why
![image](https://github.com/finity69x2/fan-percent-button-row/assets/90913570/64d08199-d878-4a3b-8308-57a39d1dfd15)

**Comment by finity69x2** [comment]
This could be caused by a few things:

have you added the card correctly into the system?

which method did you use to install it (HACS or manual)?

have you restarted HA?

Have you cleared your browser cache (ctrl-F5 in chrome)?

**Comment by mclever** [comment]
Ran into the same issue today after updating home assistant to the latest version. Had to redownload this integration and then clear browser cash for it to come back

**Comment by AndrewPDeer** [comment]
I have exactly the same problem.

1. Installed via HACS
2. Manually put a copy of "fan-control-entity-row.js" into /local/custom-lovelace/fan-control-entity-row/
3. Used the UI to add "fan-control-entity-row.js" as a resource using the path "/local/custom-lovelace/fan-control-entity-row/fan-control-entity-row.js"
4. When this didn't work I also tried adding the following to configuration.yaml manually:
```
resources:
  - url: /local/custom-lovelace/fan-control-entity-row/fan-control-entity-row.js
    type: module
```
5. If I try to add the card via the UI I get the spinning wheel (three are appearing because I've re-installed 3 times).
![image](https://github.com/user-attachments/assets/faee424d-d91c-4422-900e-f16bc1a35cd7)
6. I've restarted HA (fully), cleared the browser cache. 

I'm out of ideas now.

HA Core 2024.7.2, Supervisor 2024.06.2, Operating System 12.4, Frontend 20240710.0
